packages/provisioning/__tests__/cli/delete/instance/DeleteInstance.handler.unit.test.ts

```diff
@@ -23,6 +23,7 @@ const DEFAULT_PARAMTERS: IHandlerParameters = {
         _: ["provisioning", "delete", "instance"],
         ...UNIT_TEST_ZOSMF_PROF_OPTS
     },
+    positionals: ["provisioning", "delete", "instance"],
```

Comment:
👍  Really handy property to have (coming from `@zowe/imperative/packages/cmd/src/doc/handler/IHandlerParameters.ts`)